### DISCUSSION TITLE
Linux Client

### DISCUSSION THREAD

**Discussion by MonkeyMysteryCoding** [cmu-sei/GHOSTS#286] [Q&A]
Hello.
I am new to computers and am still in the process of learning through projects. I understood how to setup the Windows client, however, for the Linux client I am unsure on how to even start up the program. I would greatly appreciate assistance. Thanks.

**Comment by sei-dupdyke** [comment]
Hi, thanks for checking out the project and welcome to computing :)

Linux install should be straightforward:

1. Your client Linux machine will need to have [Microsoft dotnetcore runtime](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) installed. You only need the runtime installed, not the full SDK. 

2. [Download the latest Linux client](https://github.com/cmu-sei/GHOSTS/releases/latest) zip file. Unzip to a folder such as ~/ghosts for the user that you want GHOSTS to run as.

3. Now cd into that folder and run `dotnet ghosts.client.linux.dll`.
